### PR TITLE
publish job progression

### PR DESCRIPTION
**Pull request by mathiaHT**
Add a function to publish job progression and modify example according to it.

### CODE REVIEW

File: examples/process_message.rs [hunk 1]
```diff
@@ -58,7 +59,14 @@ pub fn process_message(job: &Job, job_result: JobResult) -> Result<JobResult, Me
...
 static WORKER_EVENT: WorkerEvent = WorkerEvent {};
-
+fn publish_job_progression(
+  _channel: &Channel,
+  _job: &Job,
+  progression: u8,
+) -> Result<(), lapin_futures::Error> {
+  println!("progression: {}%", progression);
+  Ok(())
+}
```
Comment (MarcAntoine-Arnaud) [inline examples/process_message.rs]: add blank line

File: examples/process_message.rs [hunk 2]
```diff
@@ -58,7 +59,14 @@ pub fn process_message(job: &Job, job_result: JobResult) -> Result<JobResult, Me
 }
 
 static WORKER_EVENT: WorkerEvent = WorkerEvent {};
-
+fn publish_job_progression(
```
Comment (MarcAntoine-Arnaud) [inline examples/process_message.rs]: add blank line before

**Comment by MarcAntoine-Arnaud** [comment]
change `publish_job_progression` return type with: `Result<(), MessageError>`
(map error into the method)

**Comment by MarcAntoine-Arnaud** [comment]
add example of progress into the `worker.rs` `process_message`

**Comment by MarcAntoine-Arnaud** [comment]
in `JobProgression` add the field `docker_container_id` generated via the method `load_docker_container_id("/proc/self/cgroup")`

**Comment by mathiaHT** [comment]
> add example of progress into the `worker.rs` `process_message`

Should I redefine a `publish_job_progression` here?

File: src/lib.rs [hunk 3]
```diff
@@ -50,6 +50,8 @@ pub trait MessageEvent {
   where
     Self: std::marker::Sized,
   {
+    
+
```
Comment (MarcAntoine-Arnaud) [inline src/lib.rs]: don't add that ...

**Pull request merged**